### ISSUE TITLE
DWMWA_WINDOW_CORNER_PREFERENCE and DWMWA_WINDOW_CORNER_RADIUS not documented

### ISSUE THREAD

**Issue by batzen** [MicrosoftDocs/windows-uwp#3425]
The documentation site refers to DWMWA_WINDOW_CORNER_PREFERENCE and DWMWA_WINDOW_CORNER_RADIUS being specified in https://docs.microsoft.com/en-us/windows/win32/api/dwmapi/ne-dwmapi-dwmwindowattribute but they are not listed there.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6cd4ed11-b246-94af-c310-514326282eaf
* Version Independent ID: 5a1dbf05-790d-525b-bcc6-01857c1687de
* Content: [Apply rounded corners in desktop apps - Windows apps](https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/apply-rounded-corners)
* Content Source: [hub/apps/desktop/modernize/apply-rounded-corners.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/hub/apps/desktop/modernize/apply-rounded-corners.md)
* Product: **uwp**
* Technology: **apps**
* GitHub Login: @jwmsft
* Microsoft Alias: **jimwalk**

**Comment by jwmsft** [comment]
@duncanmacmichael, does this article need to be updated?

**Comment by duncanmacmichael** [comment]
Hi @batzen, thanks for the feedback. DWMWA_WINDOW_CORNER_PREFERENCE was not documented previously as it was part of the Insider/prerelease builds only, so I documented its values in a table on this conceptual page. But now that it is post-release, I can update the DWMWINDOWATTRIBUTE enum to include it. 

DWMWA_WINDOW_CORNER_RADIUS was not implemented in the end, so we removed that example from the docs. Apologies for any confusion.
